pywhy_graphs/networkx/algorithms/causal/m_separation.py

```diff
@@ -21,24 +22,13 @@ def m_separated(
     """Check m-separation among 'x' and 'y' given 'z' in mixed-edge causal graph G, which may
     contain directed, bidirected, and undirected edges.
 
-    This algorithm adapts the linear time algorithm presented in [1]_ currently implemented in
-    `networkx.algorithms.d_separation` to work for mixed-edge causal graphs, using m-separation
-    logic detailed in [2]_.
-
-    This algorithm works by retaining select edges in each of the directed, bidirected, and
-    undirected edge subgraphs (if supplied). Then, an undirected graph is created from
-    the union of allsuch retained edges (without direction information), and then
-    m-separation of x and y givenz is determined if x is disconnected from y in this graph.
-
-    In the directed edge subgraph, nodes and associated edges are removed if they are childless
-    and not in x | y | z; this process is repeated until no such nodes remain. Then, outgoing
-    edges from z are removed. The remaining edges are retained.
+    This implements the m-separation algorithm TESTSEP presented in [1]_ for ancestral mixed
+    graphs, which is itself adapted from [2]_. Further checks have ensure that it works
+    for non-ancestral mixed graphs (e.g. ADMGs). The algorithm performs a breadth-first search
+    over m-connecting paths between 'x' and 'y' (i.e. a path on which every node that is a
+    collider is in 'z', and every node that is not a collider is not in 'z'). The algorithm
+    has runtime O(|E| + |V|) for number of edges |E| and number of vertices |V|.
```

Comment:
```suggestion
    This implements the m-separation algorithm TESTSEP presented in [1]_ for ancestral mixed
    graphs, which is itself adapted from [2]_. Further checks have ensure that it works
    for non-ancestral mixed graphs (e.g. ADMGs). The algorithm performs a breadth-first search
    over m-connecting paths between 'x' and 'y' (i.e. a path on which every node that is a
    collider is in 'z', and every node that is not a collider is not in 'z'). The algorithm
    has runtime ``O(|E| + |V|)`` for number of edges |E| and number of vertices ``|V|``.
```

Reply:
Is there a better way of formatting docstrings other than manually breaking lines? It's not covered by the linting and I always get messed up by it lol

Reply:
Yeah not sure. It's because if you assume it is a `py:obj` instead of a `literal` in the `conf.py`, I think they can't differentiate sometimes. I think double ` always works to bold it tho. Otw, we could render it in "math mode". Sphinx is opaque haha